native/Cargo.toml

```diff
@@ -33,21 +33,21 @@ edition = "2021"
 rust-version = "1.79"
```

Comment:
this probably can be updated to 1.80 like in datafusion or 1.81 which PR is already created